ispybLib.py

```diff
@@ -203,6 +203,8 @@ def insertResult(result,resultType,request,visitName,dc_id=None,xmlFileName=None
    sample = request['sample'] # this needs to be created and linked to a DC group
    if (resultType == 'fastDP'):
      mx_data_reduction_dict = xml_file_to_dict(xmlFileName)
+     comm = mx_data_reduction_dict['AutoProcProgramContainer']['AutoProcProgram']['processingCommandLine']
+     mx_data_reduction_dict['AutoProcProgramContainer']['AutoProcProgram']['processingCommandLine'] = comm[(len(comm)-255:]
```

Comment:
Unsure if this will work `comm[(len(comm)-255:]` doesn't look like the open parenthesis before len doesn't have a closing one

Reply:
resolved with https://github.com/NSLS-II/lsdc/pull/326/commits/5a17f0c7083aacfce9c8c17da6a04f6ac8a66a02